src/hal/getpinmap_thisboard.cpp

```diff
@@ -47,6 +47,12 @@ const HalPinmap_t *GetPinmap_ThisBoard(void)
       /* legacy names */ \
       defined(ARDUINO_CATENA_4612)
         return GetPinmap_Catena4612();
+#elif defined(ARDUINO_MCCI_CATENA_4617)
+        return GetPinmap_Catena4617();
```

Comment:
this could just use the 4612 pinmap.

Reply:
I think, We created new variant for Catena-4617 in BSP. So follow the same process. 
Can we use like
``#elif defined(ARDUINO_MCCI_CATENA_4617)
   return GetPinmap_Catena4612();``

Else, shall we create a common pinmap file for 461x.